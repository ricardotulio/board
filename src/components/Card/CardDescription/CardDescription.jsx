import React from 'react'

const CardDescription = ({ description }) => (
  <div className="card-description">
    { description }
  </div>
)

export default CardDescription
