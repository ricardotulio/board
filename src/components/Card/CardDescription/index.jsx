export { default as CardDescription } from './CardDescription.jsx'
