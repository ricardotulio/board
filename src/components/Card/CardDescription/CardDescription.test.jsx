import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import CardDescription from './CardDescription'

it('can render card description', () => {
  const description = 'it is a card description'
  const { container, getByText } = render(
    <CardDescription description={description} />
  )

  expect(getByText('it is a card description')).toBeInTheDocument()
  expect(container.firstChild).toMatchSnapshot()
})
