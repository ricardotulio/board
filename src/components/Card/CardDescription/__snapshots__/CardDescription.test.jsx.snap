// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render card description 1`] = `
<div
  class="card-description"
>
  it is a card description
</div>
`;
